src/views/UserConfirmation.vue

```diff
@@ -96,6 +97,7 @@
     props: ['partyId'],
     async ionViewWillEnter() {
       await this.store.dispatch("user/getSelectedUserDetails", { partyId: this.partyId });
+      showToast(translate("User created successfully"))
```

Comment:
This toast should be added after the createUser action is success and not after redirection.